plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintBasePlugin.kt

```diff
@@ -24,11 +27,27 @@ open class KtlintBasePlugin : Plugin<Project> {
             }
         }
 
+        val kotlinScriptAdditionalPathApplier = { fileCollection: FileCollection ->
+            val configureAction = Action<Task> {
+                with(it as KtlintCheckTask) {
+                    fileCollection.forEach {
+                        val additionalPath = project.fileTree(it)
+                        additionalPath.include("*.kts")
+                        source = source.plus(additionalPath)
+                    }
```

Comment:
Do you really need to iterate over the entire `fileCollection` here? I feel like that will just slow down the configuration phase. If there's some way to get away without iterating over the entire thing, I think that would be optimal.

Reply:
Changed to `List<String>` in extension, so applier creates a `FileTree` from provided paths.